manifests/charts/istio-control/istio-discovery/templates/validatingadmissionpolicy.yaml

```diff
@@ -3,6 +3,8 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingAdmissionPolicy
 metadata:
   name: "stable-channel-policy{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}-{{ .Values.global.istioNamespace }}.istio.io"
+  labels:
```

Comment:
Do we care about labeling nonworkload resources?

Reply:
See other comment - 

We _do_ want the "fixed" labels on every single resource.

We _do not_ probably want to apply user-defined custom labels to every single resource.

So let's just template the fixed labels, which for sure should go on everything.